fapolicy_analyzer/ui/database_admin_page.py

```diff
@@ -14,46 +14,64 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 import gi
+from fapolicy_analyzer.ui.ui_page import UIAction, UIPage
+
+gi.require_version("Gtk", "3.0")
+from gi.repository import Gtk
 
 import fapolicy_analyzer.ui.strings as strings
 from fapolicy_analyzer.ui.ancillary_trust_database_admin import (
     AncillaryTrustDatabaseAdmin,
 )
 from fapolicy_analyzer.ui.system_trust_database_admin import SystemTrustDatabaseAdmin
-from fapolicy_analyzer.ui.ui_page import UIPage
 from fapolicy_analyzer.ui.ui_widget import UIWidget
 
-gi.require_version("Gtk", "3.0")
-from gi.repository import Gtk  # isort: skip
```

Comment:
This reordering of imports is odd here. Typically its best for formatter if we keep the `from gi.repository` line at the end and keep the `# isort: skip`  so formatters to try and move it out from under the `gi.require_version` check.